src/game/savegame_legacy.c

```diff
@@ -352,6 +353,15 @@ static void Savegame_Legacy_ReadLOT(LOT_INFO *lot)
     Savegame_Legacy_Read(&lot->target, sizeof(PHD_VECTOR));
 }
 
+static void Savegame_Legacy_SetCurrentPosition(int level_num)
```

Comment:
I don't think this function is needed anymore? Loading all three types of saves seems to work without it. Need to do all thorough testing scenarios for all types of saves still, so I am leaving it in until that is confirmed.

Reply:
AFAIK it's obligatory for pickups and inventory to work with TombATI saves.